docs/user-guide/media-editor-window.md

```diff
@@ -0,0 +1,93 @@
+# Media Editor Window
+**********************
+
+The **Change media source** button allows to access the Media Editor Window:
+
+<img src="../img/media-editor-window/media-editor-window.jpg" class="ms-docimage"/>
+
+Through this window, you can add or edit three different types of media:
+
+* Images
+
+* Videos 
+
+* Maps
+
+## Images
+
+In order to add an image, the user can click on the **Images** button <img src="../img/button/images.jpg" class="ms-docbutton"/> for switch to the images section. In this section of the media editor window, with a click on the **Add** button <img src="../img/button/+++.jpg" class="ms-docbutton"/>, it is possible to define the image settings.
+
+<img src="../img/media-editor-window/img-settings.jpg" class="ms-docimage"/>
+
+In particular, it is possible to insert the following parameters:
+
+* The **Source** of the image (link to find that resource)
+
+* The **Title** of the image
+
+* The **Alternative text**, that appears if the link is broken
+
+* A **Description**, available from the image settings and in preview
+
+* The **Credits**, displayed on the bottom-right of the image
+
+!!! warning
+    **Source** and **Title** are mandatory fields.
+
+With a click on **Save** <img src="../img/button/save-icon.jpg" class="ms-docbutton"/>, the image is insert in images list and immediately available on preview.
+
+<img src="../img/media-editor-window/img-preview.jpg" class="ms-docimage"/>
+
+!!! note
+    It's always possible to change an image's settings through the **Edit** button <img src="../img/button/editing-button.jpg" class="ms-docbutton"/>.
+
+Finally, the image is inserted into the story by clicking on the **Apply** button <img src="../img/button/apply-button2.jpg" class="ms-docbutton"/>.
+
+## Maps
+
+In order to add a map, the user can click on the **Maps** button <img src="../img/button/maps-button.jpg" class="ms-docbutton"/> for swich to the map section. Clicking on the **Add** button <img src="../img/button/+++.jpg" class="ms-docbutton"/> and  the **Map Editor** window opens.
```

Comment:
In order to add a map to the story, the story editor can click on the **Maps** tab <img src="../img/button/maps-button.jpg" class="ms-docbutton"/> in order to swich to the map section of the Media Editor. Here the list of maps ready to be used for the story is available: the editor can search and select a map in the list to apply it in the story or create a map from scratch by clicking on the **Add** button <img src="../img/button/+++.jpg" class="ms-docbutton"/> to open the **Map Editor**.